### DISCUSSION TITLE
Run Project locally to start contributing

### DISCUSSION THREAD

**Discussion by brayanjuls** [mtxr/vscode-sqltools#1159] [Q&A] [answered]
I find this project interesting cause I am using it and I would like to contribute, but quite frankly I am new to typescript. I would appreciate it if someone can help me build this project on my local machine(macos). I tried to search for documentation about the development local setup but did not find it.  I tried using npm install but I am getting yarn errors. If anyone helps me I would be glad to do my first contribution by documenting the procedure of how to build the project. 

Node version: 18.12.1

Errors after executing `npm install -G` on package folder: 
```
npm ERR! code 127
npm ERR! path /Users/brayan_jules/projects/open-source/vscode-sqltools/packages/formatter
npm ERR! command failed
npm ERR! command sh -c -- yarn run build
npm ERR! sh: yarn: command not found
```

I did` npm yarn install` but i am still getting the same error.

**Comment by archiewood** [comment, marked as answer]
I just got this working.

first install yarn, and node. (it didnt work with node >=18 for me, I had to use node 16)

`npm i -g yarn`

then run 
```
yarn
yarn run start
```

Docs here: https://vscode-sqltools.mteixeira.dev/en/contributing/building
